packages/core/src/components/control-bar/control-bar.ts

```diff
@@ -0,0 +1,576 @@
+import { LitElement, html, css } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import type { VisualizationData } from "@protspace/utils";
+
+export interface ControlBarState {
+  projections: string[];
+  features: string[];
+  selectedProjection: string;
+  selectedFeature: string;
+  selectionMode: boolean;
+  isolationMode: boolean;
+  selectedProteinsCount: number;
+}
```

Comment:
Following the logic of the other components. It makes sense to move this to a interface def file as well.